### ISSUE TITLE
Fix: MXP / ROMLEX problem with "deployment number 0"

### ISSUE THREAD

**Issue by DavePearce** [Consensys/besu-sequencer-plugins#1014]
_(to recreate this requires the fix for #998 to have been applied --- hopefully this will be merged shortly)_

There is some kind of problem affecting the `fatMxp()` and `testMxpRandomAdvanced()` tests which only appears after the fix for #998.  I have marked these tests as `@Disabled` temporarily until this issue is resolved.  The trace is:

```
Exceptions triggered while tracing:
  - RomChunk with address 0x6ff1019c622e4641f86f4bb7232b7901b8d20db6, deployment number 0 and deploymentStatus true not found


java.lang.RuntimeException: RomChunk with address 0x6ff1019c622e4641f86f4bb7232b7901b8d20db6, deployment number 0 and deploymentStatus true not found
	at net.consensys.linea.zktracer.module.romlex.RomLex.getCodeFragmentIndexByMetadata(RomLex.java:99)
	at net.consensys.linea.zktracer.module.hub.fragment.account.AccountFragment.resolvePostConflation(AccountFragment.java:197)
	at net.consensys.linea.zktracer.module.hub.defer.DeferRegistry.resolvePostConflation(DeferRegistry.java:142)
	at net.consensys.linea.zktracer.module.hub.Hub.traceEndConflation(Hub.java:467)
```

@letypequividelespoubelles @lorenzogentile404 FYI

**Comment by letypequividelespoubelles** [comment]
test `testRandomMxpInstructionsFollowedByTriggeringRoob()` to reenable too

**Comment by OlivierBBB** [comment]
@DavePearce You should try my latest PR --- I've solved similar issues there, mayb this is sufficient ?

**Comment by OlivierBBB** [comment]
I am cautiously optimistic that this bug was fixed in the latest commit to #1027
